src/components/MenuCart.js

```diff
@@ -0,0 +1,50 @@
+import { files } from '../utils';
+import { Icons } from '../utils/icons';
+export function MenuCart({count,setCount}){
+    const resetCount=()=>{
+        setCount(0);
+    };
+
+    const totalPrice=125*count;
```

Comment:
move this number to a variable:


```
const DEFAULT_PRICE = 125;
const totalPrice=DEFAULT_PRICE*count;
```

check this article:

https://www.refactornow.dev/course/module-2/magic-number